packages/components/nfts/NFTView.tsx

```diff
@@ -216,46 +208,53 @@ export const NFTView: React.FC<{
                   </View>
                 )}
               </View>
-              <ImageWithTextInsert
-                size={contentWidth}
-                imageURL={nft.imageUri}
-                textInsert={nft.textInsert}
-                style={{ marginTop: 15, marginBottom: 20, borderRadius: 12 }}
-              />
-              <BrandText
-                style={{
-                  fontSize: 14,
-                  marginBottom: 12,
-                }}
-              >
-                {nft.name}
-              </BrandText>
-              <View
-                style={{
-                  flexDirection: "row",
-                  alignItems: "center",
-                  justifyContent: "space-between",
+              <OmniLink
+                to={{
+                  screen: "NFTDetail",
+                  params: { id: nft.id },
                 }}
               >
+                <ImageWithTextInsert
+                  size={contentWidth}
+                  imageURL={nft.imageUri}
+                  textInsert={nft.textInsert}
+                  style={{ marginTop: 15, marginBottom: 20, borderRadius: 12 }}
+                />
+                <BrandText
+                  style={{
+                    fontSize: 14,
+                    marginBottom: 12,
+                  }}
+                >
+                  {nft.name}
+                </BrandText>
                 <View
                   style={{
                     flexDirection: "row",
                     alignItems: "center",
-                    flex: 1,
+                    justifyContent: "space-between",
                   }}
                 >
-                  <NetworkIcon size={12} networkId={nft.networkId} />
-                  <BrandText
-                    numberOfLines={1}
+                  <View
                     style={{
-                      fontSize: 12,
-                      marginLeft: 10,
+                      flexDirection: "row",
+                      alignItems: "center",
+                      flex: 1,
                     }}
                   >
-                    {nft.collectionName}
-                  </BrandText>
+                    <NetworkIcon size={12} networkId={nft.networkId} />
+                    <BrandText
+                      numberOfLines={1}
+                      style={{
+                        fontSize: 12,
+                        marginLeft: 10,
```

Comment:
use layout.padding_x

Reply:
we don't have a 10, the closest is a 12, I will leave it as is.

Reply:
In that case I think we should respect the layout so you can use 8 or 12 instead of 10